include/cantera/zeroD/FlowDevice.h

```diff
@@ -15,9 +15,9 @@ namespace Cantera
 class Func1;
 class ReactorBase;
 
-const int MFC_Type = 1;
-const int PressureController_Type = 2;
-const int Valve_Type = 3;
+const int MFCType = 1;
+const int PressureControllerType = 2;
+const int ValveType = 3;
```

Comment:
Given the eventual goal of deprecating and removing these constants, I don't think they should be renamed. If nothing else, renaming them now would break compatibility with any code that was using these names without warning.

Reply:
understood